sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.students.controller.getAssessmentNOA.e2e-spec.ts

```diff
@@ -172,6 +173,150 @@ describe("AssessmentStudentsController(e2e)-getAssessmentNOA", () => {
       .expect(expectation);
   });
 
+  it("Should get the student NOA details for a full-time application when the application assessment is created.", async () => {
+    // Arrange
+    const enrolmentDate1 = addDays(1);
+    // Create the new student to be mocked as the authenticated one.
+    const student = await saveFakeStudent(db.dataSource);
+    // Mock user services to return the saved student.
+    await mockUserLoginInfo(appModule, student);
+    // MSFAA required for the NOA be returned.
+    const msfaaNumber = createFakeMSFAANumber(
+      { student },
+      {
+        msfaaState: MSFAAStates.Signed,
+      },
+    );
+    await db.msfaaNumber.save(msfaaNumber);
+    // Creates the applications to get the NOA.
+    const application = await saveFakeApplicationDisbursements(
+      db.dataSource,
+      {
+        msfaaNumber,
+        student,
+      },
+      {
+        offeringIntensity: OfferingIntensity.fullTime,
+      },
+    );
+    const assessmentData = {
```

Comment:
parentAssessedContribution which is used for `Parental Expected Contributions` can be added to the test data.